src/css/main.css

```diff
@@ -1,285 +1,299 @@
 html {
-  font-size: 1em;
-  line-height: 1.5;
-  height: 100%;
-  box-sizing: border-box;
+    font-size: 1em;
+    line-height: 1.5;
+    height: 100%;
+    box-sizing: border-box;
+    scroll-padding-top: 60px; /* height of sticky header */
```

Comment:
this and position sticky are actually AMAZING. Learned something new today :) nicely done!